crates/lang/src/reflect/trait_def/info.rs

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use crate::{
+    reflect::ContractEnv,
+    ToAccountId,
+};
+
```

Comment:
These are only used once, might as well just inline them at the call site

Reply:
It will looks ugly=)

```rust

pub trait CallBuilder:
    crate::reflect::ContractEnv
    + crate::ToAccountId<<Self as crate::reflect::ContractEnv>::Env>
{
}
```